FrugalApp/server.js

```diff
@@ -0,0 +1,49 @@
+// server.js
+
+// BASE SETUP
+// =============================================================================
+
+// call the packages we need
+var express    = require('express');        // call express
+var app        = express();                 // define our app using express
+var bodyParser = require('body-parser');
+
+// configure app to use bodyParser()
+// this will let us get the data from a POST
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+
+var port = process.env.PORT || 3000;        // set our port
+
+// ROUTES FOR OUR API
+// =============================================================================
+var routerGeneralInfo = express.Router();              // get an instance of the express Router
```

Comment:
Add all of this router related code should be in `app.js`. `server.js` is just a duplicate of `app.js` in this case. Move this logic into the `app.js` file and we should be good.